community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/InputIterator.java

```diff
@@ -103,6 +103,24 @@ public long position()
         {
             return actual.position();
         }
+
+        @Override
+        public int numberOfProcessors()
+        {
+            return actual.numberOfProcessors();
+        }
+
+        @Override
+        public boolean incrementNumberOfProcessors()
+        {
+            return actual.incrementNumberOfProcessors();
+        }
+
+        @Override
+        public boolean decrementNumberOfProcessors()
+        {
+            return actual.decrementNumberOfProcessors();
+        }
```

Comment:
That sounds like a good idea. Same range of behavioural freedom, but a smaller interface.